tests/prepare-wordpress.sh

```diff
@@ -29,6 +29,12 @@ for WP_SLUG in 'master' 'latest' 'previous'; do
 		;;
 	esac
 
+	clone_exit_code=$?
+	if [ $clone_exit_code -ne 0 ]; then
+		echo "Failed to clone WordPress from develop.git.wordpress.org"
+		exit -1
```

Comment:
What do you say about using `travis_terminate 1` here instead?

Reply:
We don't seem to use it anywhere. What would be the benefit ?

Reply:
I notice Travis doesn't work with so well when there's a `set +e` before the script call, but since that's not the case here, we should be safe anyhow. `travis_terminate` kinda forces it to work.